sympy/holonomic/recurrence.py

```diff
@@ -319,3 +319,36 @@ def __eq__(self, other):
                 return True
             else:
                 return False
+
+
+class HolonomicSequence(object):
+
+    def __init__(self, recurrence, u0=[]):
+        self.recurrence = recurrence
+        if not isinstance(u0, list):
+            self.u0 = [u0]
+        else:
+            self.u0 = u0
+
+        if len(self.u0) == 0:
+            self._have_init_cond = False
+        else:
+            self._have_init_cond = True
+        self.n = recurrence.parent.base.gens[0]
+
+    def __repr__(self):
+
```

Comment:
I believe so, at least if there is no docstring. PEP8 only says this:
`Use blank lines in functions, sparingly, to indicate logical sections.`